frontend/src/main/components/Shift/ShiftTable.js

```diff
@@ -0,0 +1,42 @@
+
+import OurTable from "main/components/OurTable"
+
+
+
+export default function ShiftTable({ shift }) {
+
+    // Stryker disable all : hard to test for query caching
+    // Stryker restore all 
+
+    // Stryker disable next-line all : TODO try to make a good test for this
```

Comment:
Are these lines necessary? Since your are disabling and then restoring again I don't think you really need them. Also line 11 disables something that I don't believe needs disabling.